README.md

```diff
@@ -40,6 +40,8 @@ The main commands supported by the CLI are:
 * `faas-cli deploy` - deploys the functions into a local or remote OpenFaaS gateway
 * `faas-cli remove` - removes the functions from a local or remote OpenFaaS gateway
 * `faas-cli invoke` - invokes the functions and reads from STDIN for the body of the request
+* `faas-cli login` - stores basic auth credentials for OpenFaaS gateways (multi-tenancy support)
```

Comment:
> multi-tenancy support

This isn't the "multi-tenancy support" people on the project are going to expect. Maybe - supports multiple gateways.